pkgs/development/python-modules/block-io/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchPypi, buildPythonPackage, base58, ecdsa, pycrypto, requests, six }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "block-io";
+  version = "1.1.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1sycnwpbhmzr487sc4cgj5nxvpzmn29ii122lm82avyfbbghz7g1";
+  };
+
+  patches = [
+    ./patch-pycrypto
```

Comment:
Have you submitted the patch upstream?

Reply:
I should maybe :)

Reply:
Please do that and then best use `fetchpatch` here.

Reply:
Ping @nyanloutre

Reply:
done